tests/integration/commentlog.test.js

```diff
@@ -0,0 +1,123 @@
+                    .request(app)
+                    .post(`/api/v1/articles/${slug}/comments`)
+                    .set("Authorization", `Bearer ${token}`)
+                    .send(newComment)
+                    .end((er, res) => {
+                      if (!er) {
+                        commentId = res.body.comment.id;
+                      }
+
+                      done(er || undefined);
+                    });
+                });
+            }
+          });
+      }
+    });
+});
+describe("Creating history of a comment", () => {
+  it("should return a history created", done => {
+    chai
+      .request(app)
+      .put(`/api/v1/articles/${slug}/comments/${commentId}`)
+      .set("Authorization", `Bearer ${token}`)
+      .send({ body: "Hello world" })
+      .end((error, history) => {
+        if (error) done(error);
+        expect(history.status).equals(CREATED);
+        expect(history.body.commentHistory).to.be.an("object");
+        expect(history.body.commentHistory.commentId).equals(commentId);
+        expect(history.body.commentHistory).to.haveOwnProperty("updatedAt");
+        expect(history.body.commentHistory).to.haveOwnProperty("old_version");
+        expect(history.body.commentHistory).to.haveOwnProperty("new_version");
+        done();
+      });
+  });
+  it("should return an array containing different versions", done => {
+    chai
+      .request(app)
+      .get(`/api/v1/articles/${slug}/comments/${commentId}/history`)
+      .set("Authorization", `Bearer ${token}`)
+      .end((error, history) => {
+        if (error) done(error);
+        expect(history.status).equals(OK);
+        expect(history.body.comment.commentHistory).to.be.an("array");
+        expect(history.body.comment.commentHistory[0]).to.haveOwnProperty(
+          "createdAt"
+        );
+        expect(history.body.comment.commentHistory[0]).to.haveOwnProperty(
+          "old_version"
+        );
+        expect(history.body.comment.commentHistory[0]).to.haveOwnProperty(
+          "new_version"
+        );
+        done();
+      });
+  });
+  it("should return fetching history of non-existing comment", done => {
+    chai
+      .request(app)
+      .get(`/api/v1/articles/${slug}/comments/${fakeId}/history`)
+      .set("Authorization", `Bearer ${token}`)
+      .end((error, history) => {
+        if (error) done(error);
+        expect(history.status).equals(NOT_FOUND);
+        expect(history.body.message).equals("The comment doesn't exist");
+        done();
+      });
+  });
+  it("should return unauthorized request (wrong token)", done => {
+    chai
+      .request(app)
+      .get(`/api/v1/articles/${slug}/comments/${commentId}/history`)
+      .set("Authorization", `Bearer ${token}s`)
+      .end((error, history) => {
+        if (error) done(error);
+        expect(history.status).equals(UNAUTHORIZED);
+        expect(history.body.message).equals(
+          "Please provide a token to perform this action"
```

Comment:
What do you think about this error message ?
"We are sorry but we are not able to authenticate you.You have to login to perform this action."
Instead of the message you put here?
I think the above message is more user friendly than what we have in the code.